### PR TITLE
in #merge_system_config turn truthy strings into true or false as needed

### PR DESCRIPTION
**Pull request by dbwest**

### CODE REVIEW

File: CORE/world_gadgets/configuration_engine.rb [hunk 1]
```diff
@@ -20,7 +20,9 @@ def merge_system_config
     @config_data.keys.each do |option_name|
       if ENV[option_name]
         puts "Loading value [#{ENV[option_name]}] for ENV variable [#{option_name}] from system environment."
-        @config_data[option_name] = ENV[option_name]
+        bool_map = { "true"=>true, "false"=>false }
+        option_value = bool_map.keys.include?(ENV[option_name]) ? bool_map[ENV[option_name]] : ENV[option_name]
```
Comment (greenarrowdb) [inline CORE/world_gadgets/configuration_engine.rb]: `bool_map.keys.include?(ENV[option_name]) ? ...` could just be `bool_map[ENV[option_name]] ? ...`

Comment (greenarrowdb) [reply CORE/world_gadgets/configuration_engine.rb]: Another option:
```
option_value = option_value=="true" ? true : option_value
option_value = option_value=="false" ? false : option_value
```

Comment (dbwest) [reply CORE/world_gadgets/configuration_engine.rb]: i like the second option so far best.

Comment (greenarrowdb) [reply CORE/world_gadgets/configuration_engine.rb]: Me too :)

Comment (dbwest) [reply CORE/world_gadgets/configuration_engine.rb]: i would rather add a .to_bool to String

**Pull request merged**